relay-event-normalization/src/stacktrace.rs

```diff
@@ -12,39 +12,38 @@ fn is_url(filename: &str) -> bool {
         || filename.starts_with("applewebdata:")
 }
 
-pub fn process_stacktrace(stacktrace: &mut RawStacktrace, _meta: &mut Meta) -> ProcessingResult {
+pub fn process_stacktrace(stacktrace: &mut RawStacktrace, _meta: &mut Meta) {
     // This processing is only done for non raw frames (i.e. not for exception.raw_stacktrace).
     if let Some(frames) = stacktrace.frames.value_mut() {
         for frame in frames.iter_mut() {
-            processor::apply(frame, process_non_raw_frame)?;
+            process_non_raw_frame(frame);
         }
     }
-
-    Ok(())
 }
+pub fn process_non_raw_frame(frame: &mut Annotated<Frame>) {
```

Comment:
This function remains functionally the same, its contents are now wrapped in a `processor::apply` call. The alternative is to wrap the calls to the function in a `processor::apply` that always returns `Ok(())` to satisfy the processor's API, which is unnecessarily complex (vs just a function call).